lighthouse-core/audits/seo/link-text.js

```diff
@@ -18,6 +18,10 @@ const BLOCKLIST = new Set([
   'right here',
   'more',
   'learn more',
+  'home',
+  'login',
+  'menu',
+  'contact us',
```

Comment:
hm, would you mind explaining why these link text values are a problem @MayurSidd? 

these new ones seem like legitimate descriptions of what the link will do unlike "Here" or "Go"

Reply:
Hi Patrick, 
Actually, I'm working on one MVC application to integrate with PWA. So, in the console, it's showing me an error that link description is not correct.

I see this repository in google docs for the link descriptions. So, thought that if I can add these descriptions then maybe its resolved the error. Because I tried other combinations as well.

But if you think it's not the case then can you please give me guideline or help me to resolve those link description errors?


Thanks in Advanced!

Reply:
Hm, sorry but that's not the case. Adding the strings here will result in *more* Lighthouse errors. I'm not sure what would be putting errors in your console for having a link called "Home", but it's not Lighthouse.

Perhaps you have mistaken the SEO audit for an accessibility warning?

Reply:
okay thanks will check that and try to find out the source of the problems.